org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -110,8 +110,16 @@ public String handleRequest(Map<String, String> lambdaInput, Context context) {
     final AmazonS3 s3Client = AmazonS3ClientBuilder.standard().build();
     final AWSFileManager fileManager =
         new AWSFileManager(s3Client, outputBucketName, javabuilderSessionId, getOutputUrl, context);
+    final LifecycleNotifier lifecycleNotifier = new LifecycleNotifier();
+    // TODO: Move common setup steps into CodeExecutionManager#onPreExecute
```

Comment:
Is there a task for this TODO?

Reply:
Don't think so, I'll create one

Reply:
https://codedotorg.atlassian.net/browse/CSA-1146